x-pack/plugins/reporting/server/config/create_config.test.ts

```diff
@@ -108,7 +103,25 @@ describe('Reporting server createConfig$', () => {
         },
       }
     `);
-    expect((mockLogger.warn as any).mock.calls.length).toBe(0);
+    expect(mockLogger.warn.mock.calls.length).toBe(0);
+  });
+
+  it('show warning when kibanaServer.hostName === "0.0.0.0"', async () => {
```

Comment:
Not sure this test description matches the body of the test since we are not checking for a warning being logged, rather the output of `createConfig$`

Reply:
Fixed as part of 86fd8ce10ad